ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py

```diff
@@ -1555,7 +1555,7 @@ def arrays_for_pooling(
         )
     if array_dim == 3:
         kernel = draw(st.tuples(st.integers(1, in_shape[1])))
-    new_kernel = kernel
+        new_kernel = kernel
     if return_dilation:
```

Comment:
This will actually make it fail if the array dim is not 3, it shouldn't be indented.